<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>intensDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QDoubleSpinBox" name="length">
       <property name="minimum">
        <double>0.010000000000000</double>
       </property>
       <property name="maximum">
        <double>5000.000000000000000</double>
       </property>
       <property name="value">
        <double>1064.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Відстань від лінзи, см</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="Z">
       <property name="minimum">
        <double>0.010000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="value">
        <double>15.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="F">
       <property name="minimum">
        <double>0.010000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="value">
        <double>8.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Фокусна відстань, см</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Довж. хвилі, нм</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Радіус, пучка, px (25 мкм)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QDoubleSpinBox" name="R">
       <property name="minimum">
        <double>0.010000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="value">
        <double>25.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="recalcAi2">
       <property name="text">
        <string>Обчислити</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Ai2 =</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Ai2">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QListWidget" name="namesList"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="file">
       <property name="text">
        <string>Вибрати</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="recalcCalibr">
       <property name="text">
        <string>Обчислити коеф. калібр.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_16">
     <property name="title">
      <string>Інтенсивність</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="2">
       <widget class="QLineEdit" name="calibr">
        <property name="maxLength">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="intensFilt"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Коефіцієнт калібровки:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_36">
        <property name="text">
         <string>Фільтри</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="recalcResult">
       <property name="text">
        <string>Обчислити</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Результуючий множник:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="result"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Застосувати до:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QComboBox" name="activeDataType"/>
        </item>
        <item>
         <widget class="QPushButton" name="applyForActiveData">
          <property name="text">
           <string>Застосувати</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Або:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="applyFilesList"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="getFilesForRecalc">
          <property name="text">
           <string>Файли</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="xColumn"/>
        </item>
        <item>
         <widget class="QLineEdit" name="saveSuffix">
          <property name="placeholderText">
           <string>суфікс</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="applyForFiles">
          <property name="text">
           <string>Зберегти</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QPushButton" name="hideWindow">
     <property name="text">
      <string>Закрити</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>xColumn</tabstop>
  <tabstop>applyForFiles</tabstop>
  <tabstop>length</tabstop>
  <tabstop>F</tabstop>
  <tabstop>Z</tabstop>
  <tabstop>R</tabstop>
  <tabstop>Ai2</tabstop>
  <tabstop>namesList</tabstop>
  <tabstop>file</tabstop>
  <tabstop>recalcCalibr</tabstop>
  <tabstop>calibr</tabstop>
  <tabstop>intensFilt</tabstop>
  <tabstop>result</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
